test/fixtures/recordings/Integration-test-for-up-command_3614339512/up-command-delivers-correct-response-_668345128/recording.har

```diff
@@ -37,98 +37,78 @@
               "value": "with params"
             }
           ],
-          "url": "https://adobeioruntime.net/api/v1/web/helix/default/embed/https://embeds.project-helix.io/jekyll_embed?params=with%20params"
+          "url": "https://adobeioruntime.net/api/v1/web/helix/helix-services/embed@v1https://embeds.project-helix.io/jekyll_embed?params=with%20params"
         },
         "response": {
-          "bodySize": 127,
+          "bodySize": 131,
           "content": {
-            "mimeType": "text/html; charset=UTF-8",
-            "size": 127,
-            "text": "<div class=\"embed  embed-has-title\">\n    <span class=\"title\">Fastly error: unknown domain embeds.project-helix.io\n</span>\n<div>"
+            "mimeType": "application/json",
+            "size": 131,
+            "text": "{\"code\":\"lZkz4WXIaxOAcL0iw6Ntor30zQ4axjUo\",\"error\":\"The resource requires authentication, which was not supplied with the request\"}"
```

Comment:
Hm. That doesn't seem right either.

Reply:
see above... probably dues to the missing `/`